rotkehlchen/chain/ethereum/transactions.py

```diff
@@ -16,3 +33,77 @@ def __init__(
             database: 'DBHandler',
     ) -> None:
         super().__init__(evm_inquirer=ethereum_inquirer, database=database)
+
+    def query_for_graph_delegation_txns(self, addresses: Sequence[ChecksumEvmAddress]) -> None:
+        """This function is used to query for DelegationTransferredToL2 events from thegraph
+        contract, if the events have delegated to any of the accounts. Then it will add the
+        transactions to the database."""
+        if len(addresses) == 0:
+            return None  # do nothing for no addresses
+
+        staking_contract = self.evm_inquirer.contracts.contract(CONTRACT_STAKING)
+        for address in addresses:
+            # fetch contract approval events to consider if we query the logs
+            db_filter = EvmEventFilterQuery.make(
+                counterparties=[CPT_THEGRAPH],
+                location=Location.from_chain_id(self.evm_inquirer.chain_id),
```

Comment:
This is ethereum transactions

```suggestion
                location=Location.ETHEREUM,
```